jbpm-system-test/jbpm-tx-ejb-timers-test/src/test/java/org/kie/samples/integration/testcontainers/KieServerContainer.java

```diff
@@ -45,6 +45,7 @@ public KieServerContainer(Network network, Map<String,String> args) {
     
       withEnv("START_SCRIPT", args.get("START_SCRIPT"));
       withEnv("JAVA_OPTS", "-Xms256m -Xmx2048m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=512m -Djava.net.preferIPv4Stack=true -Dfile.encoding=UTF-8 "+
+                           "-Dorg.kie.jbpm.timer.retry.limit=1 -Dorg.kie.jbpm.timer.retry.interval=500 "+
```

Comment:
is this changed related to the update to 7.61.0.Final?

Reply:
Right, basically it's due to changes introduced by https://github.com/kiegroup/jbpm/pull/2001
Instead of "PLATFORM" approach (which it's deprecated), now it follows "TIMER_RETRY_POLICY", so we have to configure just one retry in a short interval to keep the same timing in the test.